test/ActiveLogin.Authentication.BankId.AspNetCore.Test/Helpers/FakeRemoteIpAddressMiddleware.cs

```diff
@@ -0,0 +1,23 @@
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore.Test.Helpers
+{
+    internal class FakeRemoteIpAddressMiddleware
+    {
+        private readonly RequestDelegate next;
```

Comment:
Resharper tells me that suggested name is `_name` (underscore prefix).

Reply:
yup!